e2etest/config/frr/bgpd.conf

```diff
@@ -0,0 +1,43 @@
+! -*- bgp -*-
```

Comment:
There are a few typos in the commit message.

Reply:
Also, could you update it to include the reason why you need to do this as I think that is the most important thing to get across.